go.mod

```diff
@@ -18,40 +18,40 @@ require (
 	github.com/digitalocean/godo v1.93.0
 	github.com/go-ldap/ldap/v3 v3.4.4
 	github.com/go-logr/logr v1.3.0
-	github.com/google/gnostic v0.6.9
+	github.com/google/gnostic-models v0.6.8
 	github.com/google/gofuzz v1.2.0
 	github.com/hashicorp/vault/api v1.9.2
 	github.com/hashicorp/vault/sdk v0.9.0
 	github.com/kr/pretty v0.3.1
 	github.com/miekg/dns v1.1.50
-	github.com/onsi/ginkgo/v2 v2.9.5
-	github.com/onsi/gomega v1.27.7
+	github.com/onsi/ginkgo/v2 v2.13.0
+	github.com/onsi/gomega v1.29.0
 	github.com/pavlo-v-chernykh/keystore-go/v4 v4.4.1
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.15.1
-	github.com/spf13/cobra v1.7.0
+	github.com/prometheus/client_golang v1.16.0
+	github.com/spf13/cobra v1.8.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.9.0
 	golang.org/x/crypto v0.24.0
 	golang.org/x/oauth2 v0.11.0
 	golang.org/x/sync v0.7.0
 	gomodules.xyz/jsonpatch/v2 v2.3.0
 	google.golang.org/api v0.126.0
-	k8s.io/api v0.27.2
-	k8s.io/apiextensions-apiserver v0.27.2
-	k8s.io/apimachinery v0.27.2
-	k8s.io/apiserver v0.27.2
-	k8s.io/client-go v0.27.2
-	k8s.io/code-generator v0.27.2
-	k8s.io/component-base v0.27.2
-	k8s.io/klog/v2 v2.100.1
-	k8s.io/kube-aggregator v0.27.2
-	k8s.io/kube-openapi v0.0.0-20230515203736-54b630e78af5
-	k8s.io/utils v0.0.0-20230505201702-9f6742963106
+	k8s.io/api v0.29.0
```

Comment:
Consider upgrading to the latest patch version v0.29.7 of the K8S libraries, here or in a follow up PR
 * https://github.com/kubernetes/api/compare/v0.29.0...v0.29.7
 * https://github.com/kubernetes/apiextensions-apiserver/compare/v0.29.0...v0.29.7
 * https://github.com/kubernetes/apimachinery/compare/v0.29.0...v0.29.7
 * https://github.com/kubernetes/apiserver/compare/v0.29.0...v0.29.7
 * https://github.com/kubernetes/client-go/compare/v0.29.0...v0.29.7
 * etc